lcls_tools/cor_plot/cor_plot_mat_scan_test.py

```diff
@@ -0,0 +1,90 @@
+import sys
+import unittest
+import numpy as np
+from cor_plot_mat_scan import CorPlotMatScan as CPMS
+
+# We need a scan where all struct names are populated
+# to fully test.  Would have been nice to populate every
+# possible field with something (even empty array), but that
+# would be too similar to something that makes sense in the world
+# of data structures
+
+BAD_FILE = 'junk.mat'
+
+# TEST FIle metadata
+BEAM_NAMES_TEST = (
+    'profx',
+    'xStat',
+    'xStatStd',
+    'profy',
+    'yStat',
+    'yStatStd',
+    'profu',
+    'uStat',
+    'uStatStd',
+    'method',
+    'stats',
+    'statsStd'
+)
+TEST_FILE = 'test_scan.mat'
+FIELDS_TEST = (
+    'accelerator',
+    'status',
+    'ctrlPV',
+    'beam',
+    'profPV',
+    'ts',
+    'config'
+)
+ACCL_TEST = 'LCLS2'
+CTRL_PV_TEST = 'SOLN:GUNB:212:BCTRL'
+CTRL_VAL_0_TEST = 0.073  # First value of scan ctrl pv
```

Comment:
Did these constants come from the file?

Reply:
They do come from the files, it's a way of verifying that we always get the same answer based on any code changes